Cannot compile Magics 3.1.0 on CentOS: required CMake >= 3.1.0, found 2.8.12.2

See https://travis-ci.org/ARPA-SIMC/Magics-rpm/jobs/406199795#L570-L571:

```
CMake Error at CMakeLists.txt:13 (cmake_minimum_required):
  CMake 3.1.0 or higher is required.  You are running version 2.8.12.2
```

--- a/Magics.spec
+++ b/Magics.spec
@@ -1,9 +1,11 @@
 %if 0%{?rhel} == 7
 %define python3_vers python34
 %define cmake_vers cmake3
+%define ctest_vers ctest3
 %else
 %define python3_vers python3
 %define cmake_vers cmake
+%define ctest_vers ctest
 %endif
 
 %global releaseno 1
@@ -130,7 +132,7 @@ popd
 pushd build
 # MAGPLUS_HOME is needed for the tests to work, see:
 # https://software.ecmwf.int/wiki/display/MAGP/Installation+Guide
-MAGPLUS_HOME=%{buildroot} CTEST_OUTPUT_ON_FAILURE=1 LD_LIBRARY_PATH=%{buildroot}%{_libdir} ctest
+MAGPLUS_HOME=%{buildroot} CTEST_OUTPUT_ON_FAILURE=1 LD_LIBRARY_PATH=%{buildroot}%{_libdir} %{ctest_vers}
 popd
 
 %install

